### PR TITLE
New release

### PR DESCRIPTION
**Pull request by kiskoza**
We already have a few fixes merged in, we should do a release soon to make sure that this gem remains usable.

ToDo:
* [x] Set up a CI to verify the code changes
* [x] Review open PRs and merge them if they make sense
* [x] Go through the open issues and verify if they are still a problem
* [x] Write a changelog in History
* [x] Refresh the readme
* [x] Bump version number
* [ ] Make a new tag and release the new version 

----

My Questions:

**Should we add a minimum Ruby version?**

I wasn't able to set up the environment for Ruby <2.0, and there are already EOL for ages.

**Should we release it as 1.0.0 as well?**

The gem seems to be really stable, but most of the gems / projects are referencing it as `~> 0.4.x`, so we definitely need a `0.4.6` first.

### CODE REVIEW

**Comment by kiskoza** [comment]
@bf4 @jnunemaker I'm ready with all the pre-release tasks I wanted to do before we can release the new `v0.4.6` version :tada: 

I'm happy to merge in this PR and tag the new release commit in the repo any time from now. Let me know if you would like to add anything else.

Also, could you help me with releasing it to RubyGems?

File: History [hunk 1]
```diff
@@ -1,3 +1,12 @@
+== 0.4.6 2024-01-29
+
+* minor patches
+  * Ruby 3.1, 3.2, 3.3 support
+  * Ship LICENSE file
+  * Add BidDecimal to gem dependency
```
Comment (bf4) [inline History:6]:
```suggestion
  * Add BigDecimal to gem dependency
```

File: README.md [hunk 2]
```diff
@@ -1,25 +1,26 @@
 # crack
 
-[![Build Status](https://travis-ci.org/jnunemaker/crack.svg?branch=master)](https://travis-ci.org/jnunemaker/crack)
+[![Test](https://github.com/jnunemaker/crack/actions/workflows/test.yml/badge.svg)](https://github.com/jnunemaker/crack/actions/workflows/test.yml)
```
Comment (bf4) [inline README.md:3]: 👍

**Comment by bf4** [comment]
Looks good

**Comment by bf4** [comment]
Released 0.4.6

**Pull request merged**